guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -58,40 +58,14 @@ A {SmartProxy} behind a load balancer takes precedence over a {SmartProxy} selec
 ifdef::katello,satellite,orcharhino[]
 . In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
 endif::[]
-. Optional: Select the *Insecure* option, if you want to make the first call insecure.
-During this first call, hosts download the CA file from {Project}.
+. Optional: Select the *Insecure* option, if you want to make the registration call insecure.
+During an insecure call, the host downloads the CA file from {Project}.
 Hosts will use this CA file to connect to {Project} with all future calls making them secure.
 +
 {Team} recommends that you avoid insecure calls.
 +
 If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
 Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
-+
-Instead, you can manually copy and install the CA file on each host before registering the host.
-+
-To do this, find where {Project} stores the CA file by navigating to *Administer* > *Settings* > *Authentication* and locating the value of the *SSL CA file* setting.
-+
-Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on hosts and enter the following commands:
-+
-[options="nowrap", subs="+quotes,attributes"]
-----
-# update-ca-trust enable
-# update-ca-trust
-----
```

Comment:
My main point was to move this into an action that has to be done as a prerequisite to registration so that the user can make a secure call right away.

Reply:
As long as Foreman Web UI offers the Insecure checkbox, we should keep the procedure on how to import the Foreman CA cert. Instead of deleting it, maybe move it to a separate procedure which we could specify as prerequisite?